src/components/CourseForm/CourseForm.tsx

```diff
@@ -62,111 +77,180 @@ export default function CourseForm({ courseData }: Props) {
+            id="courseFormDescription"
+            multiline
+            minRows={5}
+            error={!!errors.description}
+            autoComplete="off"
+            inputProps={{
+              'data-testid': 'courseFormDescription',
+              style: {
+                resize: 'block',
+              },
+            }}
+          />
+          <FormFieldError error={errors.description} />
 
-        <InputLabel htmlFor="courseFormStartDate">Start date</InputLabel>
-        <Input
-          {...register('startDate')}
-          defaultValue={
-            courseData ? dateToDateTimeLocal(courseData.startDate) : ''
-          }
-          id="courseFormStartDate"
-          type="datetime-local"
-          error={!!errors.startDate}
-          inputProps={{
-            'data-testid': 'courseFormStartDate',
-          }}
-        />
-        <FormFieldError error={errors.startDate} />
+          <Controller
+            name="tags"
+            control={control}
+            defaultValue={[]}
+            render={({ field }) => {
+              return (
+                <>
+                  <InputLabel htmlFor="tagSelection">Tags</InputLabel>
+                  <Select
+                    {...field}
+                    id="tagSelection"
+                    multiple
+                    renderValue={(field) => (
+                      <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 0.5 }}>
+                        {field.map((tag, idx) => (
+                          <Chip
+                            key={idx}
+                            label={tag}
+                            variant="outlined"
+                            sx={{
+                              backgroundColor: palette.secondary.light,
+                              borderColor: palette.black.light,
+                            }}
+                          />
+                        ))}
+                      </Box>
+                    )}
+                  >
+                    {tags.map((tag) => (
+                      <MenuItem
+                        key={tag.id}
+                        value={tag.name}
+                        divider
+                        sx={{
+                          '&.Mui-selected.Mui-focusVisible': {
+                            backgroundColor: palette.secondary.dark,
+                          },
+                          '&:hover': {
+                            backgroundColor: palette.secondary.light,
+                          },
+                          '&:focus': {
+                            backgroundColor: palette.secondary.main,
+                          },
+                          '&.Mui-selected': {
+                            backgroundColor: palette.secondary.main,
+                          },
+                          '&.Mui-selected:hover': {
+                            backgroundColor: palette.secondary.main,
+                          },
+                        }}
```

Comment:
```suggestion
                       sx={{
                          '&.Mui-selected': {
                            backgroundColor: palette.secondary.main,
                          },
                          '&.Mui-selected.Mui-focusVisible': {
                            backgroundColor: palette.secondary.dark,
                          },
                          '&:hover': {
                            backgroundColor: palette.secondary.light,
                          },
                          '&.Mui-selected:hover': {
                            backgroundColor: palette.secondary.main,
                          },                         
                        }}
```

Which browser / OS did you test that btw? Windows with Firefox and Chrome doesn't behave like it should when deselecting items in the select menu. Basically when you try remove an item it still stays in darker shade. The style snippet I provided works for me at least.

Reply:
Ubuntu 22.04 / Firefox 118.0.1 it seemed to work fine. There is some quirky behavior after deselecting all tags using arrow keys / enter, where the top-most tag will stay highlighted even after closing and re-opening the menu, which is what the `&:focus` was trying to fix. Besides that niche issue, the style snippet you provided seems to be working better. Will fix!